### ISSUE TITLE
Profiler instructions from the doc produce an error when installing pmutools

### ISSUE THREAD

**Issue by michaelpaper** [ease-lab/vhive#440]
I am trying to use VHive's profiler.
I followed each step of the quick start guide (cloning version 1.4.1) on a CloudLab cluster with the "vHive-cluster-env" profile with 2 nodes.
Then I followed the steps described in `docs/profiling.md`, but when running `scripts/install_pmutools.sh`, I get the following error:

```
Traceback (most recent call last):
  File "/usr/local/pmu-tools/toplev", line 3781, in <module>
    version = runner_emaps()
  File "/usr/local/pmu-tools/toplev", line 3771, in runner_emaps
    runner.ectx.emap = ocperf.find_emap(pmu=runner.pmu if runner.pmu else "cpu")
  File "/usr/local/pmu-tools/ocperf.py", line 979, in find_emap
    return json_with_extra(el, eventmap_is_file, pmu)
  File "/usr/local/pmu-tools/ocperf.py", line 877, in json_with_extra
    name = event_download.eventlist_name(el, pmu_to_type[pmu])
  File "/usr/local/pmu-tools/event_download.py", line 232, in eventlist_name
    files = parse_map_file(name, key, acceptfile=True)
  File "/usr/local/pmu-tools/event_download.py", line 146, in parse_map_file
    cpu, version, name, typ = n
ValueError: too many values to unpack
```

Do you know why this happens and how to fix it ?

Thanks in advance !

**Comment by ustiugov** [comment]
thanks for reporting this issue. @amohoste also noticed the same error few days ago

@dhschall as you are an active user of PMU tools, could you please take a look at why it fails in our script?

**Comment by dhschall** [comment]
Hey, @michaelpaper 
yes I also encountered this problem and it seems to come from a bug in the upstream repo. 
I fixed it by adding a `, *_` in the tuple unpacking of [line 146 ](https://github.com/andikleen/pmu-tools/blob/93a31782131f907067339c883477075cfedb5451/event_download.py#L146)

**Comment by ustiugov** [comment]
@dhschall could you please submit a bug fix to upstream?

**Comment by dhschall** [comment]
Created [issue](https://github.com/andikleen/pmu-tools/issues/408#issue-1065941426) and pull request in upstream repo.

**Comment by dhschall** [comment]
I pushed now my changes also to our own [fork](https://github.com/ease-lab/pmu-tools/pull/1)

**Comment by ustiugov** [comment]
@dhschall please open a PR in this repo as well, changing the branch and the repo to the ones in our fork.

**Comment by ustiugov** [comment]
fixed by #445
